spec/integration/manifest.spec.yml

```
@@ -363,6 +362,20 @@ spec:
       $ref: "#/definitions/categories"
     conditions:
       $ref: "#/definitions/conditions"
+    # requires a conditional JSON schema to update the value depending
+    # on the policy_templates length
+    policy_templates_behavior:
+      description: >
+        Behavior to manage the policy templates (policy_templates field) defined that must be available in Kibana as tiles.
+        This is just applicable for those packages that define more than one policy template. By default for these packages, Kibana
+        shows all the policy templates defined in the manifest separately plus another one referring to the
+        whole integration (that contains all the policy templates definitions) available to the user.
+
+        If the `integration_package_only` option is selected, then just one policy template for the whole integration containing all policy
+        templates will be available in Kibana. If the `policy_templates_only` option is selected, then just the policy templates defined
+        in the manifest will be available in Kibana without the specific policy template for the whole integration. If the `default` option
+        is selected, then Kibana keeps the same behavior described above.
```

Simplified wording, removing references to the visual representation. Also proposing these three values:
* `combined_policy`: to indicate that there is a single policy template that combines them all.
* `individual_policies`: to indicate that policies are available individually, without a combined policy.
* `all`: the default value, where both the combined policy and the individual policies are available.

```suggestion
        Expected behavior when there are more than one policy template defined.
        When set to `combined_policy`, a single policy template is available, that 
        combines all the defined templates. When set to `individual_policies`, all
        policies are individually available, but there is no combined policy.
        The default value is `all`, where the combined policy template is available
        along with the individual policies.
```

---

I like this proposal that does not mention any reference to tiles in Kibana.
And it also looks easier to understand what each option is aimed for.

Thanks! 

I'll do the required changes.

---

Added in https://github.com/elastic/package-spec/pull/825/commits/b2ff18906289256c01d80e8e585175bf433e92d5